tests/integration/tui/screenshots/test_auto_expansion.later-time.html

```diff
@@ -0,0 +1,21 @@
+<pre><span style="color:#ff0000;background:#e5e5e5;font-weight:bold">C</span><span style="color:#ffff00;background:#e5e5e5;font-weight:bold">y</span><span style="color:#00ff00;background:#e5e5e5;font-weight:bold">l</span><span style="color:#5c5cff;background:#e5e5e5;font-weight:bold">c</span><span style="color:#000000;background:#e5e5e5;font-weight:bold"> Tui</span><span style="color:#7f7f7f;background:#e5e5e5">   workflows filtered (</span><span style="color:#7f7f7f;background:#e5e5e5;font-weight:bold">W</span><span style="color:#7f7f7f;background:#e5e5e5"> - edit, </span><span style="color:#7f7f7f;background:#e5e5e5;font-weight:bold">E</span><span style="color:#7f7f7f;background:#e5e5e5"> - reset)</span><span style="color:#000000;background:#e5e5e5">                             </span>
```

Comment:
Are you OK with this offest?

![Untitled](https://github.com/cylc/cylc-flow/assets/26465611/8f9d9f76-4f33-455d-a16c-163663fbf522)


(it appears in lots of your images)

Reply:
The reason for this is the way your browser handles the queued modifier.

In XTerm, the modifier applies to the task icon with a character width of 1. In FireFox the modifier comes before the task icon with a character width of 2.